include/tc/c2/tc_op.h

```diff
@@ -37,11 +37,12 @@ class TcOp : public Operator<Context> {
  public:
   TcOp(const OperatorDef& operator_def, Workspace* ws)
       : caffe2::Operator<Context>(operator_def, ws),
+        tc_(OperatorBase::GetSingleArgument<std::string>("lang", "ERROR")),
```

Comment:
here the `lang` argument is read

Reply:
@ftynse def is a keyword in python, that can't be used. I am happy to use "tc" though if you like

Reply:
yes, I am still making changes, thanks for catching this :)